src/main/java/bio/terra/service/filedata/FileService.java

```diff
@@ -113,6 +115,10 @@ public String ingestFile(
 
   public String ingestBulkFile(
       String datasetId, BulkLoadRequestModel loadModel, AuthenticatedUserRequest userReq) {
+    if (loadModel.isBulkMode() && StringUtils.isEmpty(loadModel.getLoadTag())) {
```

Comment:
I think this logic probably should live in the `bio.terra.service.dataset.IngestRequestValidator` class?